lib/osf-components/addon/components/banners/view-only-link/template.hbs

```diff
@@ -13,9 +13,10 @@
             data-test-view-normally
             data-analytics-name='Stop view-only link'
             @type='default'
-            @onClick={{action this.stopViewOnly}}
+            {{on "click" this.stopViewOnly event=click}}
+            {{on "click" @manager.cancel event=click}}
```

Comment:
These should be single quotes. Also, I don't see this `event` argument used anywhere on `stopViewOnly` action? Where's the `@manager` coming from here?